resources/europe/be/be-matrix.json

```diff
@@ -7,7 +7,6 @@
   "name": "OpenStreetMap BE Riot channel",
   "description": "All mappers are welcome! Sign up at {signupUrl}",
```

Comment:
I you remove `signupUrl` you also should remove `Sign up at {signupUrl}` from here.

Reply:
Fixed, tx for feedback